src/Activation/useActivationNavigation.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  determineActivationSteps,
+  Environment,
+} from "./useActivationNavigation"
+
+describe("useActivationNavigation", () => {
```

Comment:
It might be helpful for future readers to have these specs be more granular.

```suggestion
describe("determineActivationSteps", () => {
  describe("when the locationPermissions are required and off", () => {
     it("includes the activate location step", () => {
     
   describe("when bluetooth is not enabled", () => {
     it("includes the enable bluetooth step", () => {
     
   etc...
```

Reply:
Sounds good!